ansible/library/extract_log.py

```diff
@@ -77,11 +77,18 @@
 import gzip
 import re
 import sys
+import logging
 from datetime import datetime
 from functools import cmp_to_key
 from ansible.module_utils.basic import *
 
 
+def config_logging():
```

Comment:
Would it be safer to add custom logger with buffer stream to the log file instead of calling `basicConfig`?

Reply:
changed.